### ISSUE TITLE
Documentation for user executable scripts

### ISSUE THREAD

**Issue by kpinc** [Pylons/pyramid#1153]
It would be nice if pserve, and whatever other scripts are supposed to be executed by the user, were documented.  I'm thinking of a reference document similar to the API document.  Although I don't think the API docs are indexed and it'd be nice to have index entries for the scripts.

I imagine that the  -h output would be sufficient, unless it's not.

**Comment by stevepiercy** [comment]
I got this working locally. I'll add index entries for each p* script.

Before I start work, I'd like some feedback on the following two items.

1. I am leaning toward creating separate pages, one for each p* script, similar to the API pages. This will make it easy to find and reference throughout the documentation.

    ```rst
    `p*` Scripts Documentation
    ==========================

    `p*` scripts included with :app:`Pyramid`:.

    .. toctree::
       :maxdepth: 1
       :glob:

       pscripts/index
       pscripts/*
    ```

2. In [Command-Line Pyramid](http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/commandline.html), I am debating whether to add links to each script page as a "See Also", or to display the output of `--help` for each script inline, or both.

**Comment by stevepiercy** [comment]
Found an issue in Sphinx 1.3.1 and logged to https://github.com/sphinx-doc/sphinx/issues/2189

**Comment by mmerickel** [comment]
If we are going to document the exact script APIs then I would put them on a separate appendix/api page. Not in the scripting docs themselves.

**Comment by stevepiercy** [comment]
I worked on this this morning and created a separate page for each p* script out the output of `p* --help`  and a link to its respective narrative docs section as the page content. I also provided links from the narrative docs to the pscripts/* pages. I think that's what @mmerickel is saying. PR will be up in a bit for review, discussion, and correction as needed.

**Comment by stevepiercy** [comment]
PR #2191 might need an addition to [CHANGES.txt](https://github.com/Pylons/pyramid/blob/master/CHANGES.txt#L303), too.

Feedback welcome.

**Comment by stevepiercy** [comment]
Preview the changes here:
http://pyramid-stevepiercy.readthedocs.org/en/master/#p-scripts-documentation

**Comment by mmerickel** [comment]
Nice @stevepiercy. Let's call it CLI (or Script?) Documentation as it shows up below API Documentation.

**Comment by stevepiercy** [comment]
Closed by #2191
